tests/api/v1/collectors/get.js

```diff
@@ -75,7 +75,7 @@ describe('tests/api/v1/collectors/get.js >', () => {
   });
 
   it('get with limit ok', (done) => {
-    const c2 = u.toCreate;
+    const c2 = JSON.parse(JSON.stringify(u.toCreate));
```

Comment:
I think it's cleaner to put the parse+stringify logic in the utils module in a "getCollectorToCreate()" fn--that way every test which uses it doesn't have to repeat the parse+stringify pattern